nixos/modules/services/networking/haproxy.nix

```diff
@@ -37,19 +37,36 @@ with lib;
       message = "You must provide services.haproxy.config.";
     }];
 
-    systemd.services.haproxy = {
```

Comment:
Could we move this to an override in our code base? This will be really hard to find when we upgrade.